{
  "comments": [
    {
      "key": {
        "uuid": "5ad8a22d_a2e14733",
        "filename": "test/testrunner/testrunner.py",
        "patchSetId": 4
      },
      "lineNbr": 837,
      "author": {
        "id": 1022530
      },
      "writtenOn": "2017-05-09T19:05:15Z",
      "side": 1,
      "message": "Would it make sense to exit or at least warn if the match fails?",
      "revId": "e09231fca0b4e1105a6c8bf87aa3e8a6002e4a28",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": false
    }
  ]
}